src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -34,6 +34,7 @@
 		<PackageReference Include="NUnitLite" Version="3.6.1" />
 		<PackageReference Include="log4net" Version="2.0.8" />
 		<PackageReference Include="Serilog" Version="2.0.0" />
+		<PackageReference Include="Serilog.Sinks.TextWriter" Version="2.1.0" />
```

Comment:
This is only defined and used in the tests. I can downgrade it if you want. It was a shameless copy/paste from Jeremy's branch.